packages/design-system/src/styles/components/_Dialog.scss

```diff
@@ -2,6 +2,53 @@
 
 $dialog-spacer: $spacer-4;
 
+// Start polyfill styles
+dialog {
```

Comment:
I inlined the polyfill CSS as discussed in #1685 but am somewhat concerned about impacting the cascade, mainly because I can't guarantee when this CSS will load in relation to styles for Drawer.

A better spot might be to add this in a CSS higher up the cascade, like an internal DS reset. Do we have something like that?

Reply:
We have control over the order because we import these files explicitly in the components index, and they're alphabetical. However, I don't think it'll matter because we use `.ds-c-dialog*` and `.ds-c-drawer*` selectors, which have a higher specificity than the `dialog` type selector. Hmm, I guess there's the `dialog.fixed` further down 🤔 . I don't actually see the `fixed` class referenced anywhere in the polyfill code. Do you know what that's about?

If we don't need to care about order, we could possibly include this in its own `_NativeDialog.scss` file.